ui/app/AppLayouts/Chat/ContactsColumn.qml

```diff
@@ -11,26 +11,19 @@ import "./CommunityComponents"
 
 import StatusQ.Core 0.1
 import StatusQ.Core.Theme 0.1
+import StatusQ.Controls 0.1
 import StatusQ.Components 0.1
 import StatusQ.Popups 0.1
 
 Item {
     id: contactsColumn
     property int chatGroupsListViewCount: channelList.chatListItems.count
-    property alias searchStr: searchBox.text
+    property alias searchStr: searchInput.text
     signal openProfileClicked()
 
     Layout.fillHeight: true
     width: 304
 
-    MouseArea {
-        anchors.fill: parent
-        onClicked: {
-            //steal focus from search field
-            addChat.forceActiveFocus();
-        }
-    }
```

Comment:
I think we needed this to remove the focus from the search box

Reply:
Figured that, but it didn't seem to work. 
OTOH you also lose focus on the input when you click a chat item or the chat button or pretty much any other focussable component, so I personally doubt we need to keep this around.

Can add it back though if people experience a strong usability issue here

Reply:
Ah I wonder if it got broken in master. I know it was @alexandraB99 who implemented this at first. I don't know who raised the issue first, but maybe @anastasiyaig would know. 
Context: do we care if the search box doesn't always lose focus (when clicking un-focusable stuff)